third_party/pcsc-lite/naclport/server/src/google_smart_card_pcsc_lite_server/global.cc

```diff
@@ -153,4 +182,37 @@ void InitializeAndRunPcscLiteServer() {
       "successfully finished.";
 }
 
+const PcscLiteServerGlobal* PcscLiteServerGlobal::GetInstance() {
+  GOOGLE_SMART_CARD_CHECK(g_pcsc_lite_server);
+  return g_pcsc_lite_server;
+}
+
+void PcscLiteServerGlobal::PostMessage(
```

Comment:
nit: Reorder the method definition order to match the order of declaration in the header file.